mllib/src/main/scala/org/apache/spark/ml/feature/MinMaxScaler.scala

```diff
@@ -111,7 +111,7 @@ class MinMaxScaler @Since("1.5.0") (@Since("1.5.0") override val uid: String)
 
   @Since("2.0.0")
   override def fit(dataset: Dataset[_]): MinMaxScalerModel = {
-    transformSchema(dataset.schema, logging = true)
+    transformSchema(dataset.schema)
```

Comment:
Seems the `transformSchema(schema: StructType, logging: Boolean)` method of the base class `PipelineStage` would call the the overloaded `transformSchema` method without the `logging` param:

https://github.com/apache/spark/blob/v2.0.0/mllib/src/main/scala/org/apache/spark/ml/Pipeline.scala#L70